crates/orderbook/src/api/get_total_surplus.rs

```diff
@@ -0,0 +1,19 @@
+use {
+    primitive_types::H160,
+    serde_json::json,
+    std::convert::Infallible,
+    warp::{http::StatusCode, reply::with_status, Filter, Rejection},
+};
+
+pub fn get() -> impl Filter<Extract = (super::ApiReply,), Error = Rejection> + Clone {
+    warp::path!("v1" / "users" / H160 / "total-surplus")
```

Comment:
Looks like we use underscores for the other endpoints.
```suggestion
    warp::path!("v1" / "users" / H160 / "total_surplus")
```

Reply:
Ah yes, nice catch.